### PR TITLE
Focus area pages

### PR DESCRIPTION
**Pull request by ZeeshaanMaudar**

### CODE REVIEW

File: packages/webapp/src/components/Preview/Heading/CustomizedSelect/index.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+} from './styled';
+
+
+const callMenuItems = ({ id, name }) => (
+  <MenuItem value={id}>{name}</MenuItem>
+);
+
+class CustomizedSelect extends Component {
+
+  state = {
```
Comment (schalkventer) [inline packages/webapp/src/components/Preview/Heading/CustomizedSelect/index.jsx]: Please change this, CRA does not support ES7.

File: packages/webapp/src/components/Preview/Heading/CustomizedSelect/index.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    budgetValue: ''
+  };
+
+  handleChange = event => {
```
Comment (schalkventer) [inline packages/webapp/src/components/Preview/Heading/CustomizedSelect/index.jsx]: Same as above.

File: packages/webapp/src/components/Preview/Heading/index.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  <Link href={url}>
+    <ButtonDetails>
+      <ButtonText>Detailed Analysis</ButtonText>
+      <ArrowStyled />
+    </ButtonDetails>
+  </Link>
+);
+
+const Heading = ({ government, departmentNames, selected, eventHandler, year, sphere }) => {
+  const url = `/${year}/${sphere}/${government === 'south-africa' ? '' : `${government}/`}departments/${selected}`;
```
Comment (schalkventer) [inline packages/webapp/src/components/Preview/Heading/index.jsx]: This is getting a bit hard to read, perhaps break it out into a function called `provinceFolder`.

Comment (schalkventer) [reply packages/webapp/src/components/Preview/Heading/index.jsx]: @ZeeshaanMaudar `const provinceFolder = government === 'south-africa' ? '' : \`${government}/\``

Comment (ZeeshaanMaudar) [reply packages/webapp/src/components/Preview/Heading/index.jsx]: Done and replaced with provinceFolder inside of url

**Pull request merged**